Sources/Sentry/SentryHub.m

```diff
@@ -539,17 +540,53 @@ - (void)configureScope:(void (^)(SentryScope *scope))callback
  */
 - (BOOL)isIntegrationInstalled:(Class)integrationClass
 {
-    for (id<SentryIntegrationProtocol> item in self.installedIntegrations) {
-        if ([item isKindOfClass:integrationClass]) {
-            return YES;
+    @synchronized(_integrationsLock) {
+        for (id<SentryIntegrationProtocol> item in _installedIntegrations) {
+            if ([item isKindOfClass:integrationClass]) {
+                return YES;
+            }
         }
+        return NO;
```

Comment:
l: can we do like this?
```suggestion
       return [_installedIntegrationNames contains:NSStringFromClass(integrationClass)];
```

Reply:
Excellent point, but sadly no. The `_installedIntegrations` are `NSArray<id<SentryIntegrationProtocol>>` and we are searching for a `Class`. So we need to use `isKindOfClass`.

Reply:
Well, this is already merged and its work.
Just for clarification, my suggestion was to use '_installedIntegrationNames' instead of '_installedIntegrations', but thats fine.

Reply:
Ah, sorry about that. Yes, that might work. But as you said, it's already merged. Sorry.